pyogrio/_compat.py

```diff
@@ -47,6 +53,8 @@
 GDAL_GE_38 = __gdal_version__ >= (3, 8, 0)
 GDAL_GE_311 = __gdal_version__ >= (3, 11, 0)
 
+GDAL_HAS_PARQUET = "Parquet" in list_drivers()
```

Comment:
Can you move this to somewhere in the tests? (since this is only used there, no need to pay the overhead of listing all drivers here in the main library import, I think?)